src/v/kafka/CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ v_cc_library(
     server/protocol_utils.cc
     server/quota_manager.cc
     server/client_quota_translator.cc
+    server/monitor_deprecated_client_quota_configs.cc
```

Comment:
Why not build this into `quota_manager`?  Why do we need a separate service for monitoring?

Reply:
I've integrated this into `client_quota_translator` now